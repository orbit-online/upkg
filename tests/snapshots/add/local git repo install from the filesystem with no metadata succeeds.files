[drwxr-xr-x]  .
├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .bin
│   │   └── [lrwxrwxrwx]  acme-empty-v1.0.2.bin -> ../.packages/acme-empty-v1.0.2-no-metadata.git@ea82f0b9fbce993deb1f969a2472d14b077e8842/bin/acme-empty-v1.0.2.bin
│   ├── [drwxr-xr-x]  .packages
│   │   └── [drwxr-xr-x]  acme-empty-v1.0.2-no-metadata.git@ea82f0b9fbce993deb1f969a2472d14b077e8842
│   │       └── [drwxr-xr-x]  bin
│   │           └── [-rwxr-xr-x]  acme-empty-v1.0.2.bin
│   └── [lrwxrwxrwx]  acme-empty-v1.0.2-no-metadata.git -> .packages/acme-empty-v1.0.2-no-metadata.git@ea82f0b9fbce993deb1f969a2472d14b077e8842
└── [-rw-r--r--]  upkg.json
