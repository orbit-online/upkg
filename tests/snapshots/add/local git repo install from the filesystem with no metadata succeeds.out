upkg: Cloning $BATS_RUN_TMPDIR/package-fixtures/acme-empty-v1.0.2-no-metadata.git
upkg: Added '$BATS_RUN_TMPDIR/package-fixtures/acme-empty-v1.0.2-no-metadata.git'
